src/qt/guiutil.cpp

```diff
@@ -671,14 +671,23 @@ QString ConnectionTypeToQString(ConnectionType conn_type, bool prepend_direction
 {
     QString prefix;
     if (prepend_direction) {
-        prefix = (conn_type == ConnectionType::INBOUND) ? QObject::tr("Inbound") : QObject::tr("Outbound") + " ";
+        prefix = (conn_type == ConnectionType::INBOUND) ?
+                     //: An Inbound Connection from a Peer.
+                     QObject::tr("Inbound") :
+                     //: An Outbound Connection to a Peer.
+                     QObject::tr("Outbound") + " ";
     }
     switch (conn_type) {
     case ConnectionType::INBOUND: return prefix;
+    //: Peer connection type which relays all network information.
     case ConnectionType::OUTBOUND_FULL_RELAY: return prefix + QObject::tr("Full Relay");
+    //: Peer connection type which only relays Block network information.
```

Comment:
throughout: s/Block/block/ as it is not a proper noun, unless part of one like "Outbound Block Relay", e.g. a connection type